apps/condo/domains/meter/tasks/sendVerificationDateReminder.spec.js

```diff
@@ -5,104 +5,122 @@
 const dayjs = require('dayjs')
 
 const { setFakeClientMode } = require('@condo/keystone/test.utils')
-const { sendVerificationDateReminder } = require('@condo/domains/meter/tasks/sendVerificationDateReminder')
-const { Message: MessageApi } = require('@condo/domains/notification/utils/serverSchema')
+
+const { sendVerificationDateReminder, sendVerificationDateReminderTaskFn } = require('@condo/domains/meter/tasks/sendVerificationDateReminder')
+
 const { METER_VERIFICATION_DATE_REMINDER_TYPE } = require('@condo/domains/notification/constants/constants')
+const { Message: MessageApi } = require('@condo/domains/notification/utils/serverSchema')
+
 const { makeClientWithResidentAndMeter } = require('../utils/testSchema')
 
 const index = require('@app/condo/index')
+
 const { keystone } = index
 
 const getNotificationsFromMeter = async ({ verificationDate, nextVerificationDate, searchWindowDaysShift = 0 }) => {
     const { user: { id } } = await makeClientWithResidentAndMeter({ verificationDate, nextVerificationDate })
+
     await sendVerificationDateReminder({ date: null, searchWindowDaysShift, daysCount: 30 })
+
     return await MessageApi.getAll(keystone, { user: { id }, type: METER_VERIFICATION_DATE_REMINDER_TYPE })
 }
 
 describe('Meter verification notification', () => {
     setFakeClientMode(index)
 
-    it('should not send messages on null nextVerificationDate', async () => {
-        const messages = await getNotificationsFromMeter({
-            verificationDate: null,
-            nextVerificationDate: null,
+    describe('Direct function tests', () => {
+        it('should not send messages on null nextVerificationDate', async () => {
+            const messages = await getNotificationsFromMeter({
```

Comment:
You need a test if verificationDate is set and nextVerificationDate is null

Reply:
done